suites/pacific/cephfs/tier-2_cephfs_test-nfs_ha.yaml

```diff
@@ -137,3 +137,15 @@ tests:
       desc: nfs-ha-failover
       polarion-id: CEPH-83575196
       abort-on-fail: false
+  - test:
+      name: nfs-ha-node-poweroff
+      module: cephfs_nfs.test_cephfs_nfs_ha_active_power_off.py
+      desc: nfs-ha-failover
+      polarion-id: CEPH-83575195
+      abort-on-fail: false
+  - test:
+      name: nfs-ha-node-replacement
+      module: cephfs_nfs.test_cephfs_nfs_ha_node_replacement.py
+      desc: nfs-ha-failover
+      polarion-id: CEPH-83575096
```

Comment:
Please add Automation script path to polarion TC field 'Automation script' and mark as Automated in 83575096

Reply:
Lets mark the test automated only after PR merge.